software/digiread.yml

```diff
@@ -0,0 +1,11 @@
+name: Digiread
+website_url: https://ladigitale.dev/digiread/
+description: A simple application to clean up online pages and articles using Mozilla's Readability (documentation in French).
```

Comment:
```suggestion
description: Clean up online pages and articles using Mozilla's Readability (documentation in French).
```

Reply:
Ha, we had the same description. beat you to it :P